marumi/ec.beginner/mypage.php

```diff
@@ -18,25 +58,28 @@
 
 </head>
 <body>
-    <div class="p-myPage__background js-login">
-    </div> <!--p-myPage__background js-login-->
-
-    <div class="p-myPage  js-login">
-        <div class="p-myPage__closeIcon js-close" type="button">
-            <span class="p-closeIcon__bar"></span>
-            <span class="p-closeIcon__bar"></span>
-        </div> <!--p-myPage__closeIcon js-close-->
-        <p class="c-section__title p-myPage__title">LOGIN</p>
-            <p><?php isset($error) ? escape($error): ''; ?></p>
-            <form class="p-myPage__wrap" action="login.php" method="post">
-                <label class="p-login__label">ユーザー名</label>
-                <input type="text" id="name" name="name" class="p-login__input" required>
-                <label class="p-login__label">パスワード</label>
-                <input type="password" id="password" name="password" class="p-login__input" required>
-                <input type="submit" class="c-btn p-login__btn" value="ログイン">
-            </form> <!--p-myPage__wrap-->
-        <a href="/signUp.php" class="p-myPage__register">新規登録はこちら</a>
-    </div> <!--p-myPage  js-login-->
+    <?php require('header.php'); ?>
+    <section class="l-mypage">
+            <div class="l-mypage__inner">
+                <h2 class="c-section__title">MY PAGE</h2>
+                <div class="p-mypage__form">
+                    <dl class="p-mypage__list">
+                        <dd class="p-mypage__name"><?php echo escape($currentUser['name']); ?></dd>
+                        <dt>メールアドレス</dt>
+                        <dd><?php echo escape($currentUser['email']); ?></dd>
+                        <dt>生年月日</dt>
+                        <dd><?php echo isset($currentUser['birthday']) ? escape($currentUser['birthday']) : ''; ?></dd>
+                        <dt>性別</dt>
+                        <dd><?php echo escape($displayGender); ?></dd>
+                    </dl>
+                
+                    <a href="#" class="c-btn p-edit__btn">メールアドレスを変更する</a>
+                    <a href="mypage_edit.php" class="c-btn p-edit__btn">その他の会員情報を変更する</a>
```

Comment:
[imo]
ここですが、一般的に編集ページを表示するときはこんな感じでパラメーターをつけてidをサーバー側へ渡してあげることが多いです。（クエリストリングやクエリパラメーターといいます）
`?key=value`の形ですね。[参考](https://online.dhw.co.jp/kuritama/query-string/)
```suggestion
                    <a href="#" class="c-btn p-edit__btn">メールアドレスを変更する</a>
                    <a href="mypage_edit.php?id=<?= $user['id'] ?>" class="c-btn p-edit__btn">その他の会員情報を変更する</a>
```

`$user['id'] `は↑で書いた `$user = $stmt->fetch();`を使ってます。

そして、サーバー側で受け取ったidでDBからレコードを取得して該当ユーザーの情報を編集ページで表示するって流れになります。
今回の場合なら`my_edit.php`でやる感じですかね 🤔